### DISCUSSION TITLE
Confirmation before leaving Add Bean page after entering data

### DISCUSSION THREAD

**Discussion by FalconPDX** [graphefruit/Beanconqueror#587] [Ideas]
I'm quite new to the app so this is coming from someone who hasn't had a lot of experience with BC. I've been greatly enjoying the app! There has been one flow that frustrated me, for which I have a suggestion as to improve it.

When switching to the Variety Information tab in the Add Bean page, you lose sight of the tabs when you scroll down. However, there is an arrow pointing to the left that stays visible. My first instinct was to press it to return to the general tab, but it ended up closing the Add Bean page, losing all of the data I had entered. Frustrated, I re-entered the data, and then when finished entering in the Variety Information I pressed the arrow again! Luckily, third time was the charm but I had to remind myself not to press the arrow.

I feel that maybe this experience could be improved by adding a dialog box, confirming that the user is about to discard the entered data when they press the back arrow. I've included screenshots to highlight the flow.

<img src="https://github.com/graphefruit/Beanconqueror/assets/7719978/7562bd41-06d9-48d4-8072-fd97887864ec" height="500">

<img src="https://github.com/graphefruit/Beanconqueror/assets/7719978/435832c6-7175-4528-8303-e1dc4a7eedc3" height="500">

<img src="https://github.com/graphefruit/Beanconqueror/assets/7719978/25c11c49-b50d-4e60-86a8-745108a574e2" height="500">

**Comment by graphefruit** [comment]
This has been supported with 7.0.0, and need to be activated in the settings.
